index.html

```diff
@@ -0,0 +1,239 @@
+            responsibilities authors have when developing accessible web
+            experiences.
+          </p>
+          <a href="/fundamentals/names-and-descriptions/" aria-label="View names and descriptions guide">Learn More</a>
+          <img
+            alt="Indicators delve inside a document."
+            src="img/index-4.svg"
+          />
+        </li>
+        <li>
+          <h3>And So Much More...</h3>
+          <p>
+            Learn about other fundamental accessibility resources and understand
+            how to Provide Accessible Names and Descriptions, implement
+            accessible Grid and Table properties, use Hiding Semantics, and
+            more.
+          </p>
+          <a class="button-link" href="/fundamentals/" aria-label="View fundamentals">Learn More</a>
+          <img
+            alt="A box with an accessibility label is chock full of widgets and document bits."
+            src="img/index-5.svg"
+          />
+        </li>
+      </ul>
+    </div>
+
+    <div id="work">
+      <h2>Current Work</h2>
+      <h3>Check out our most recent work</h3>
+      <p>
+        Review the overall scope of work planned for the APG and supporting
+        deliverables and the timelines for completing our work.
+      </p>
+      <ul>
+        <li>
+          <h3>Release Plans</h3>
+          <p>
+            Take a look at the plan overview for WAI-ARIA Authoring Practices,
+            priorities and work in progress.
+          </p>
+          <a
+            rel="noopener noreferrer"
+            target="_blank"
+            href="https://github.com/w3c/aria-practices/wiki/Release-Plans"
+            aria-label="View release plans wiki"
+          >
+            Learn More
+          </a>
+        </li>
+        <li>
+          <h3>Patterns Progress Status</h3>
+          <p>
+            Status of work on design patterns and reference implementations of
+            those patterns for the first release of the ARIA 1.1 Authoring
+            Practices.
+          </p>
+          <a
+            rel="noopener noreferrer"
+            target="_blank"
+            href="https://github.com/w3c/aria-practices/wiki/Design-Patterns-Development-Status"
+            aria-label="View pattern status wiki"
+          >
+            Learn More
+          </a>
+        </li>
+        <li>
+          <h3>How to Write Regression Tests</h3>
+          <p>
+            Learn what the APG Regression Tests test, how to run them and
+            understand the results.
+          </p>
+          <a
+            rel="noopener noreferrer"
+            target="_blank"
+            href="https://github.com/w3c/aria-practices/wiki/Regression-Tests-for-APG-Example-Pages"
+            aria-label="View regression test documentation"
+          >
+            Learn More
+          </a>
+        </li>
+        <li>
+          <h3>Meetings</h3>
+          <p>
+            The APG Task Force meets on Tuesdays at 14:00 until 15:00 Boston
```

Comment:
Perhaps an optional observation which could be addressed another time but wondering if `Boston time` conveys the same relative meaning to everyone as they would with GMT/UTC specified times? 

Perhaps we could also use the language in meeting agendas and invites, which state `Eastern Time`?

Reply:
This is a good point, that kind of stuck out to me too. I've snuck in that change.